src/components/Tabs/TabsComponent.tsx

```diff
@@ -53,6 +69,33 @@ export default function TabsComponent({ mode, gameData, setGameData }: Props) {
                 <p>{gameData.description}</p>
               )}
             </TabPanel>
+            {gameData.videoTrailer && gameData.videoTrailer !== "" ? (
+              <TabPanel className="flex flex-col justify-center">
+                <ReactPlayer
+                  url={gameData.videoTrailer}
+                  controls={true}
+                  width={1300}
+                  height={731}
```

Comment:
instead of a defined width and height, can it match the tab's width? make sure it fills and shrinks to all desktop screen sizes (try using style instead of width/height -- not sure if that would work but just a suggestion)

Reply:
can we also take advantage of [react-player/lazy](https://www.npmjs.com/package/react-player#:~:text=If%20your%20build%20system%20supports%20import()%20statements%2C%20use%20react%2Dplayer/lazy%20to%20lazy%20load%20the%20appropriate%20player%20for%20the%20url%20you%20pass%20in.%20This%20adds%20several%20reactPlayer%20chunks%20to%20your%20output%2C%20but%20reduces%20your%20main%20bundle%20size.)?